src/random.cpp

```diff
@@ -485,6 +488,14 @@ static void SeedPeriodic(CSHA512& hasher, RNGState& rng)
     // High-precision timestamp
     SeedTimestamp(hasher);
 
+    {
```

Comment:
If you'd abstract this code block out, and then call it from say both `SeedPeriodic` and `SeedSlow`, the additional entropy would always be available to strong RNG output, even if a minute has not yet passed. You could make the hashing step conditional on `events_hasher.Size() != 0`.

Reply:
Hmm, you mean in SeedSlow Finalize the events_hasher, then write the buffer back into the events_hasher and have SeedPeriodic wipe it? Cause SeedSlow doesn't store the data back into the RNGState.

Reply:
All the Seed* functions just add to a CSHA512; ultimately, they're all called from ProcRand, which mixes it with the existing RNGState, and produces output.

SeedPeriodic goes a step further, in trying to make sure that the existing RNGState is also fed through the strengthener, but that isn't really required.

Reply:
Right, so I did this.